zookeeper/src/main/java/com/linecorp/armeria/server/zookeeper/ZooKeeperUpdatingListener.java

```diff
@@ -46,13 +46,31 @@ public ZooKeeperUpdatingListener(String zkConnectionStr, String zNodePath, int s
                                      Endpoint endpoint) {
         this.zkConnectionStr = requireNonNull(zkConnectionStr, "zkConnectionStr");
         this.zNodePath = requireNonNull(zNodePath, "zNodePath");
-        this.endpoint = requireNonNull(endpoint, "endpoint");
+        this.endpoint = endpoint;
```

Comment:
then the second constructor could not use the first, cause some annoying repeated variable initialization  codes